fuel-bft/README.md

```diff
@@ -1,8 +0,0 @@
-# fuel-bft
```

Comment:
Will using a symlink here actually work on crates.io?

Reply:
[It does](https://github.com/serde-rs/serde/blob/master/serde_derive/README.md)